### DISCUSSION TITLE
VIP用户但是每日签到只给10000额度

### DISCUSSION THREAD

**Discussion by zyrmj0212** [BurnHair/docs#192] [分享讨论]
您好，我是vip用户，但是现在每天签到还是只能得10000额度，请问这是怎么回事呢？

**Comment by BennyThink** [comment]
网页签到是10000机器人才是30000

**Reply by 93814690** [reply]
> 网页签到是10000机器人才是30000

但你的定价页不是这么写的

**Reply by BennyThink** [reply]
价格按照 US$1 = 50 0000 额度计算，签到额度与签到方式相关

还有这行字 你可能没看到

**Reply by nov12** [reply]
原来如此，可是为啥更偏向tg。增加网页端日活岂不更好？tg就一个机器人。

**Reply by BennyThink** [reply]
你猜😂
